File: java/class4/RandomNumbersInLoops.java
import java.util.Random;

public class RandomNumbersInLoops {

    public static void main (String[] args) {
        
        Random rand = new Random();
        
        int random_value = rand.nextInt(10); // [0,9]
        
        System.out.println("\nRandom value [0,9]: " + random_value);
        
        System.out.println("\n20 Random Values [1,10]:");
        System.out.print("\t");
        for (int i = 0; i < 20; i++) { // 0 .. 19   (so 20 loops total)
            random_value = (rand.nextInt(10) + 1); // [1,10]
             System.out.print(random_value + " ");
        }
        System.out.println();
    }

}

File: python/class4/random_numbers_in_loops.py
import random

random_value = random.randint(1,10) # [1,10]

print "\nRandom value [1,10]: " + str(random_value)

print "\n20 Random Values [1,10]:"
for i in range(20): # 0 .. 19   (so 20 loops total)
    random_value = random.randint(1,10)
    print random_value # won't print in a single line
    
print 


### to print in a single line
#print "\n20 Random Values [1,10]:"
#random_values = [] # store values in a list/array
#for i in range(20):
#    random_values.append(random.randint(1,10))
#    
#print "\t" + str(random_values)
